Bottom: 6f580585b890f1e7d06f2095bdde0381872c8bde
Top:    84a3a7edbaffc0477060f48598f7fa21e840c8de
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-05-01 01:19:35 +0100

Refresh of alternative-more-thorough-step

---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index e98449a..6879a98 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -50,7 +50,7 @@ presentation_style_assignments_from_appearance (step_file *step, appearance *app
 }
 
 static step_file *
-start_ap214_file (char *filename)
+start_ap214_file (const char *filename)
 {
   FILE *f;
   time_t currenttime;
